pkg/policy/rule.go

```diff
@@ -421,6 +421,33 @@ func mergeIngress(policyCtx PolicyContext, ctx *SearchContext, fromEndpoints api
 		}
 		return nil
 	})
+	if err != nil {
+		return 0, err
```

Comment:
Should this be `return found, err` for consistency with the previous code in this function?